### ISSUE TITLE
We can't make a proper organisation because the name git-bug is already taken on Github

### ISSUE THREAD

**Issue by MichaelMure** [MichaelMure/git-bug#416]
Hello @git-bug

This is an attempt to contact you as you don't have any other way of contact on your Github profile.

As you can see in this repository, `git-bug` is a distributed bug tracker embedded in git. It is now relatively popular and I would like to promote it to a real Github organization to grow a community and regroup the various related repository.

The obvious `git-bug` name is taken by your account that you haven't used in years now. Would you consider releasing this name? That would be very nice of you. It's not much but I can send you some stickers of this project in exchange.

Could you contact me at batolettre@gmail.com ?

**Comment by tbm** [comment]
Inactive accounts are [against GitHub policy](https://docs.github.com/en/github/site-policy/github-username-policy). I've had success in the past asking GitHub to release (i.e. remove) an inactive account.

**Comment by MichaelMure** [comment]
I tried that already without success. I suppose it's because this account actually had an activity (2 issues years ago).

**Comment by sudoforge** [comment]
Just to note, there's no activity _that you can see_. For all we know, the user makes 100 commits a day -- if all of the activity is private, the activity panel will look empty.

**Comment by MichaelMure** [comment]
I made https://github.com/git-bug-project but there is actually no way to accept github sponsors as an org, which limit quite a lot the usefulness it.

**Comment by sudoforge** [comment]
The *Sponsors* feature is GA for organization in the following regions: https://github.com/sponsors#regions

**Comment by xloem** [comment]
@MichaelMure The @git-bug organisation is available to register now.

**Comment by MichaelMure** [comment]
@xloem thanks for the heads up ! https://github.com/git-bug

**Comment by MichaelMure** [comment]
I'm quite curious how that happened.

**Comment by theWebalyst** [comment]
Great news!

@tbm @MichaelMure how did you request release of an inactive name? There's one I'd like because I use it in lots of places which has been inactive for several years.

**Comment by MichaelMure** [comment]
@theWebalyst you can ask the github support but don't get your hopes too high, they probably won't do it if there has been any activity on that account before.

**Comment by xloem** [comment]
I took the time to check, but I don't know how it happened. 
I was guessing the owner released it after eventually noticing some communication from someone.

**Comment by 6543** [comment]
https://github.com/contact/report-abuse?report=git-bug  @MichaelMure ?

**Comment by MichaelMure** [comment]
I did that a while back but Github refused ¯\\\_(ツ)_/¯

**Comment by theWebalyst** [comment]
I've submitted s report for another account which has been inactive for almost six years, let's see what happens.

**Comment by xloem** [comment]
Just clarifying that michaelmure @'d the account, which probably left an old email in the owner's inbox.

**Comment by happybeing** [comment]
Bingo, I'm now @happybeing on github as well as many other places (and have @theWebalyst to stop impersonations)!

I'm not sure what this breaks!

**Comment by sudoforge** [comment]
Are there plans to migrate the source of truth for this project to the @git-bug organization?

**Comment by MichaelMure** [comment]
@sudoforge eventually I suppose. The main goal would be to leverage github's donation system to help build a community of contributor. At the moment, https://opencollective.com/git-bug does that job but using github's one as well would probably help.

Note that I can't benefit myself from those donations as that would put me in a gray area regarding my current day job.

**Comment by zarelit** [comment]
[just triaging] [This page](https://github.com/git-bug) now shows the git-bug logo and it is an organization so I guess this issue can be closed?

**Comment by 6543** [comment]
who owns it?
<https://github.com/orgs/git-bug/people>

**Comment by sudoforge** [comment]
Membership visibility is set to Private by default when joining or creating an organization, however, the context provided by @MichaelMure's earlier reply indicates that they have the Owner role for the organization.

**Comment by MichaelMure** [comment]
Closing as I do have control of the `git-bug` github org, ready to be used when the time comes.

**Comment by sudoforge** [comment]
> when the time comes

hey michael -- is there anything you're still waiting on before making this change?
